pkg/image/image.go

```diff
@@ -43,3 +44,56 @@ func ImageInfoFromFile(registry registry.RegistryOptions, nameParts []string) (k
 
 	return image, nil
 }
+
+func DestImageName(registry registry.RegistryOptions, srcImage string) string {
+	imageParts := strings.Split(srcImage, "/")
+	lastPart := imageParts[len(imageParts)-1] // last part
+
+	image := fmt.Sprintf("%s/%s/%s", registry.Endpoint, registry.Namespace, lastPart)
+
+	return image
+}
+
+func buildImageAlts(destRegistry registry.RegistryOptions, image string) ([]kustomizeimage.Image, error) {
```

Comment:
could you add a unit test for this function? what if the image is `library/redis` or `replicated/kots`? does it work without the domain?

Reply:
Can do
and I'll add some code to handle two-part images (since it currently handles 1 and 3+ part images)

Reply:
that code ended up being rather wrong
resolved now, though